### DISCUSSION TITLE
Semantic Kernel Office Hours

### DISCUSSION THREAD

**Discussion by Rvelazquez45** [microsoft/semantic-kernel#9290] [Office Hours Recaps]
I downloaded the office hours link from this blog https://devblogs.microsoft.com/semantic-kernel/updates-on-ways-to-engage-with-the-semantic-kernel-team/ and joined the call on 10/16/24 at 10 AM CST but there was no meeting going on. Is there a new link or has something else changed? I was joining the calls regularly until a few months ago and would like to join again.

**Comment by sophialagerkranspandey** [comment]
Hi @Rvelazquez45, we have updated links here: https://devblogs.microsoft.com/semantic-kernel/semantic-kernel-office-hours-recordings/ our calls are on Wednesday from 8-9AM (Pacific time), we look forward to hearing from you next Wednesday.
